scripts/collections.cdc

```diff
@@ -124,6 +125,23 @@ pub fun main(address: Address) : {String : MetadataCollection}? {
 	}
 	*/
 
+	let rareRoomCollection = account.getCapability<&RareRooms_NFT.Collection{RareRooms_NFT.RareRooms_NFTCollectionPublic}>(RareRooms_NFT.CollectionPublicPath).borrow()
+		?? panic("Couldn't get collection")
+
```

Comment:
The only thing I didn't do similarly to the others is to include this `if rareRoomCollection.check() {` .
With the `.borrow() ?? panic(...)` I think it's OK, but please let me know

Reply:
You cannot panic here. You can remove the panic and wrap the rest if the code in if rareRoomCollection!= nil

Reply:
If you panic it will fail for anybody who do not own a rare room nft.